week-1/Homework/J-functions/exercise.js

```diff
@@ -1,7 +1,11 @@
-function halve(number) {
-  // complete the function here
```

Comment:
Review [this exercise](https://github.com/Migracode-Barcelona/javascript-module-1/tree/main/week-1/Homework/J-functions#exercise), it expects you to define a function called `halve` that returns half the input and test it with the code already provided below:
```
const result = halve(12);
console.log(result); // 6
```